### ISSUE TITLE
Packaging Error

### ISSUE THREAD

**Issue by Tim4DConstruct** [iTwin/itwin-unreal-plugin#31]
I have been trying to package a level with this new plugin but I keep getting errors. Ill post the Output log here maybe someone can help.

LogUObjectHash: Compacting FUObjectHashTables data took   0.42ms
LogTurnkeySupport: Project requires temp target (ITwinForUnreal plugin is enabled)
LogLauncherProfile: Unable to use promoted target - ../../../../../../Users/TimvanNuland/Documents/Unreal Projects/Itwin_Plugin/Binaries/Win64/UnrealGame.target does not exist.
LogMonitoredProcess: Running Serialized UAT: [ cmd.exe /c ""C:/Program Files/Epic Games/UE_5.3/Engine/Build/BatchFiles/RunUAT.bat"  -ScriptsForProject="C:/Users/TimvanNuland/Documents/Unreal Projects/Itwin_Plugin/Itwin_Plugin.uproject" Turnkey -command=VerifySdk -platform=Win64 -UpdateIfNeeded -EditorIO -EditorIOPort=55970  -project="C:/Users/TimvanNuland/Documents/Unreal Projects/Itwin_Plugin/Itwin_Plugin.uproject" BuildCookRun -nop4 -utf8output -nocompileeditor -skipbuildeditor -cook  -project="C:/Users/TimvanNuland/Documents/Unreal Projects/Itwin_Plugin/Itwin_Plugin.uproject"  -unrealexe="C
:\Program Files\Epic Games\UE_5.3\Engine\Binaries\Win64\UnrealEditor-Cmd.exe" -platform=Win64 -installed -stage -archive -package -build -pak -iostore -compressed -prereqs -archivedirectory="C:/Users/TimvanNuland/Documents/Itwin Test" -clientconfig=Development" -nocompile -nocompileuat ]
UATHelper: Packaging (Windows): Running AutomationTool...
UATHelper: Packaging (Windows): Using bundled DotNet SDK version: 6.0.302
UATHelper: Packaging (Windows): Starting AutomationTool...
UATHelper: Packaging (Windows): Parsing command line: -ScriptsForProject="C:/Users/TimvanNuland/Documents/Unreal Projects/Itwin_Plugin/Itwin_Plugin.uproject" Turnkey -command=VerifySdk -platform=Win64 -UpdateIfNeeded -EditorIO -EditorIOPort=55970 -project="C:/Users/TimvanNuland/Documents/Unreal Projects/Itwin_Plugin/Itwin_Plugin.uproject" BuildCookRun -nop4 -utf8output -nocompileeditor -skipbuildeditor -cook -project="C:/Users/TimvanNuland/Documents/Unreal Projects/Itwin_Plugin/Itwin_Plugin.uproject" -unrealexe="C:\Program Files\Epic Games\UE_5.3\Engine\Binaries\Win64\UnrealEditor-Cmd.exe" -pl
atform=Win64 -installed -stage -archive -package -build -pak -iostore -compressed -prereqs -archivedirectory="C:/Users/TimvanNuland/Documents/Itwin Test" -clientconfig=Development -nocompile -nocompileuat
UATHelper: Packaging (Windows): Initializing script modules...
UATHelper: Packaging (Windows): Total script module initialization time: 0.13 s.
UATHelper: Packaging (Windows): Executing commands...
UATHelper: Packaging (Windows): Installed Sdk validity:
UATHelper: Packaging (Windows): Win64: (Status=Valid, MinAllowed_Sdk=10.0.00000.0, MaxAllowed_Sdk=10.9.99999.0, Current_Sdk=10.0.22621.0, Allowed_AutoSdk=10.0.18362.0, Current_AutoSdk=, Flags="InstalledSdk_ValidVersionExists")
UATHelper: Packaging (Windows): Scanning for envvar changes...
UATHelper: Packaging (Windows): ... done! 
UATHelper: Packaging (Windows): Cleaning Temp Paths...
UATHelper: Packaging (Windows): BUILD SUCCESSFUL
UATHelper: Packaging (Windows): Setting up ProjectParams for C:\Users\TimvanNuland\Documents\Unreal Projects\Itwin_Plugin\Itwin_Plugin.uproject
UATHelper: Packaging (Windows): ********** BUILD COMMAND STARTED **********
UATHelper: Packaging (Windows): Running: C:\Program Files\Epic Games\UE_5.3\Engine\Binaries\ThirdParty\DotNet\6.0.302\windows\dotnet.exe "C:\Program Files\Epic Games\UE_5.3\Engine\Binaries\DotNET\UnrealBuildTool\UnrealBuildTool.dll" Itwin_Plugin Win64 Development -Project="C:\Users\TimvanNuland\Documents\Unreal Projects\Itwin_Plugin\Itwin_Plugin.uproject" -Manifest="C:\Users\TimvanNuland\Documents\Unreal Projects\Itwin_Plugin\Intermediate\Build\Manifest.xml"  -remoteini="C:\Users\TimvanNuland\Documents\Unreal Projects\Itwin_Plugin"  -skipdeploy  -log="C:\Users\TimvanNuland\AppData\Roaming\Unre
al Engine\AutomationTool\Logs\C+Program+Files+Epic+Games+UE_5.3\UBT-Itwin_Plugin-Win64-Development.txt"
UATHelper: Packaging (Windows): Log file: C:\Users\TimvanNuland\AppData\Roaming\Unreal Engine\AutomationTool\Logs\C+Program+Files+Epic+Games+UE_5.3\UBT-Itwin_Plugin-Win64-Development.txt
UATHelper: Packaging (Windows): Creating makefile for Itwin_Plugin (manifest 'C:\Users\TimvanNuland\Documents\Unreal Projects\Itwin_Plugin\Intermediate\Build\Manifest.xml' not found)
UATHelper: Packaging (Windows): Writing manifest to C:\Users\TimvanNuland\Documents\Unreal Projects\Itwin_Plugin\Intermediate\Build\Manifest.xml
UATHelper: Packaging (Windows): Building Itwin_Plugin...
UATHelper: Packaging (Windows): Using Visual Studio 2022 14.36.32542 toolchain (C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.36.32532) and Windows 10.0.22621.0 SDK (C:\Program Files (x86)\Windows Kits\10).
UATHelper: Packaging (Windows): [Upgrade]
UATHelper: Packaging (Windows): [Upgrade] Using backward-compatible build settings. The latest version of UE sets the following values by default, which may require code changes:
UATHelper: Packaging (Windows): [Upgrade]     CppStandard = CppStandardVersion.Default        => Updates C++ Standard to C++20 (Previously: CppStandardVersion.Cpp17).
UATHelper: Packaging (Windows): [Upgrade]     WindowsPlatform.bStrictConformanceMode = true   => Updates MSVC strict conformance mode to true (Previously: false).
UATHelper: Packaging (Windows): [Upgrade] Suppress this message by setting 'DefaultBuildSettings = BuildSettingsVersion.V4;' in Itwin_Plugin.Target.cs, and explicitly overriding settings that differ from the new defaults.
UATHelper: Packaging (Windows): [Upgrade]
UATHelper: Packaging (Windows): Determining max actions to execute in parallel (24 physical cores, 32 logical cores)
UATHelper: Packaging (Windows):   Executing up to 24 processes, one per physical core
UATHelper: Packaging (Windows): ------ Building 2 action(s) started ------
UATHelper: Packaging (Windows): [1/2] Link [x64] Itwin_Plugin.exe
UATHelper: Packaging (Windows):    Creating library C:\Users\TimvanNuland\Documents\Unreal Projects\Itwin_Plugin\Binaries\Win64\Itwin_Plugin.lib and object C:\Users\TimvanNuland\Documents\Unreal Projects\Itwin_Plugin\Binaries\Win64\Itwin_Plugin.exp
UATHelper: Packaging (Windows): spdlog.lib(async.obj) : error LNK2019: unresolved external symbol _Cnd_timedwait_for referenced in function "public: bool __cdecl std::condition_variable::wait_for<__int64,struct std::ratio<1,1000>,class <lambda_f596761ae2bdb7f673f1a3ad47018eba> >(class std::unique_lock<class std::mutex> &,class std::chrono::duration<__int64,struct std::ratio<1,1000> > const &,class <lambda_f596761ae2bdb7f673f1a3ad47018eba>)" (??$wait_for@_JU?$ratio@$00$0DOI@@std@@V<lambda_f596761ae2bdb7f673f1a3ad47018eba>@@@condition_variable@std@@QEAA_NAEAV?$unique_lock@Vmutex@std@@@1@AEBV?$du
ration@_JU?$ratio@$00$0DOI@@std@@@chrono@1@V<lambda_f596761ae2bdb7f673f1a3ad47018eba>@@@Z)
UATHelper: Packaging (Windows): cpr.lib(threadpool.cpp.obj) : error LNK2001: unresolved external symbol _Cnd_timedwait_for
UATHelper: Packaging (Windows): assert.lib(stringification.cpp.obj) : error LNK2019: unresolved external symbol __std_bitset_to_string_1 referenced in function "class std::basic_ostream<char,struct std::char_traits<char> > & __cdecl std::operator<<<char,struct std::char_traits<char>,32>(class std::basic_ostream<char,struct std::char_traits<char> > &,class std::bitset<32> const &)" (??$?6DU?$char_traits@D@std@@$0CA@@std@@YAAEAV?$basic_ostream@DU?$char_traits@D@std@@@0@AEAV10@AEBV?$bitset@$0CA@@0@@Z)
UATHelper: Packaging (Windows): assert.lib(assert.cpp.obj) : error LNK2019: unresolved external symbol __std_min_8u referenced in function "auto __cdecl std::_Min_vectorized<unsigned __int64 const >(unsigned __int64 const * const,unsigned __int64 const * const)" (??$_Min_vectorized@$$CB_K@std@@YA?A_PQEB_K0@Z)
UATHelper: Packaging (Windows): C:\Users\TimvanNuland\Documents\Unreal Projects\Itwin_Plugin\Binaries\Win64\Itwin_Plugin.exe : fatal error LNK1120: 3 unresolved externals
UATHelper: Packaging (Windows): Total time in Parallel executor: 4.96 seconds
UATHelper: Packaging (Windows): Total execution time: 6.11 seconds
UATHelper: Packaging (Windows): Took 6,20s to run dotnet.exe, ExitCode=6
UATHelper: Packaging (Windows): UnrealBuildTool failed. See log for more details. (C:\Users\TimvanNuland\AppData\Roaming\Unreal Engine\AutomationTool\Logs\C+Program+Files+Epic+Games+UE_5.3\UBT-Itwin_Plugin-Win64-Development.txt)
UATHelper: Packaging (Windows): AutomationTool executed for 0h 0m 7s
UATHelper: Packaging (Windows): AutomationTool exiting with ExitCode=6 (6)
UATHelper: Packaging (Windows): BUILD FAILED

**Comment by AlexandreWalensa** [comment]
Hi Tim,
These issues should be fixed with the latest version of the Visual Studio 2022 compiler.
You are currently compiling with VS 2022 17.6, so please do the following:
- update VS 2022 to latest version (17.12)
- force UnrealBuildTool to use the latest version you installed: open the file "%appdata%\Unreal Engine\UnrealBuildTool\BuildConfiguration.xml" in a text editor and add this block inside the root <Configuration> tags:
```
<WindowsPlatform>
<Compiler>VisualStudio2022</Compiler>
<CompilerVersion>Latest</CompilerVersion>
</WindowsPlatform>
```

**Comment by Tim4DConstruct** [comment]
Thank you for the information

**Comment by GhisBntly** [comment]
Hi, I will close this issue assuming it is fixed.
Feel free to reopen if needed, or open a new issue if there are new problems with the latest version 0.1.14
Cheers,
